lib/DnD/Droppable.js

```diff
@@ -0,0 +1,30 @@
+import { useDrop } from 'react-dnd';
+import { func, node } from 'prop-types';
+
+function Droppable({ children, acceptDragTypes, onDrop, canDropChecker }) {
+  const [collected, defineDropRef] = useDrop({
+    accept: acceptDragTypes,
+    drop: onDrop,
+    canDrop: canDropChecker,
+    collect: monitor => ({
```

Comment:
I quite like being specific with what we collect. If you ever needed a different implementation then you can always use the hook in your component 👍

Reply:
I'll assume that we'll add / alter this as we use it in the hierarchy view.

Reply:
We will most likely need hover in here too, we use that heavily across the app